woeip/apps/air_quality/tests/test_dustrak.py

```diff
@@ -14,15 +13,15 @@
 @pytest.fixture
 def target_data():
     """Load the target output"""
-    target_data = pd.read_csv(test_data_directory / 'joined.csv', parse_dates=['time'])
-    target_data.set_index('time', inplace=True)
-    target_data = target_data.tz_localize('UTC')
-    target_data.reset_index(inplace=True)
+    target_output = pd.read_csv(test_data_directory / 'joined.csv', parse_dates=['time'])
+    target_output.set_index('time', inplace=True)
+    target_output = target_output.tz_localize('UTC')
+    target_output.reset_index(inplace=True)
 
-    return target_data
+    return target_output
 
 
-def test_joiner(target_data):
+def test_joiner(target_data):  # pylint: disable=W0621
```

Comment:
Is there a good time to use fixtures? I thought this was an appropriate place to use a fixture -- to avoid running the same code twice.

Reply:
From his comment on the other PR:

> This data is only needed by the one test, so there is no need to make a fixture (which you might share with multiple test modules).

Reply:
@clintonb @r-b-g-b If everyone is good with the other two changes (removing the save command and changing the assert in the test_save), can I merge this into the r-b-g-b/add-joiner? We'll have just the one PR. We can remove the fixture from that one.

Reply:
The data is needed by two tests though, ``test_joiner`` and ``test_save``?

Reply:
But yeah all good for merging!

Reply:
Most fixtures live in a separate file and are share by multiple test files. As we see, it’s just a function. Since it isn’t being used elsewhere there is no need to create a fixture.